LoadFolders.xml

```diff
@@ -0,0 +1,8 @@
+<loadFolders>
```

Comment:
The game seems to work fine without this file even in 1.2. I wonder if adding this without a 1.2 definition will break 1.2 though.

Reply:
If a version doesn’t have a matching loadFolder entry, it’ll default to the most recent version that does have an entry. So a 1.2 update would use the 1.1 loadFolder by default